### PR TITLE
Playing audio out of a MP3 file

### PR DESCRIPTION
**Pull request by Janix4000**

### CODE REVIEW

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+])
+```
+
+## Installation
+
+To run this demo one needs to install native dependencies:
+
+1. [MP3 MAD](https://github.com/membraneframework/membrane_mp3_mad_plugin/tree/v0.14.0#installation)
+2. [AAC FDK](https://github.com/membraneframework/membrane_aac_fdk_plugin#installation)
+3. [SWResample FFmpeg](https://github.com/membraneframework/membrane_ffmpeg_swresample_plugin#installation)
```
Comment (WojciechBarczynski) [inline livebook_examples/playing_mp3_file/playing_mp3_file.livemd:23]: I think that the `Installation` section should appear before the dependencies setup. This way, the user should be aware, of why something went wrong.  Moreover, the mad plugin has a poor description in the installation section, so to improve experience of this notebook users, we should either list here all required commands for installing native dependencies (on Mac and Linux) or/and create a PR improving mad plugin readme.

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  File,
+  MP3,
+  FFmpeg,
+  RawAudio,
+  AAC,
+  Kino
+}
+
+structure =
+  child(:file_source, %File.Source{location: "./fixtures/sample.mp3"})
```
Comment (WojciechBarczynski) [inline livebook_examples/playing_mp3_file/playing_mp3_file.livemd:61]: Usually we define such constants as
```elixir
@audio_path "./fixtures/sample.mp3"
```
and use
```elixir
  child(:file_source, %File.Source{location: @audio_path})
```

Comment (WojciechBarczynski) [inline livebook_examples/playing_mp3_file/playing_mp3_file.livemd:61]: I guess the copyright of `Paper Navy: Swan Song` somehow conflicts with our license. I'm not a lawyer, but I would change that for something without any copyrights reserved (royalty free).

But the song itself is quite nice :)

**Review by WojciechBarczynski — CHANGES_REQUESTED** [review]

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd (hunk 1 above)
Comment (Janix4000) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:23]: I am afraid it is obligatory to put the dependencies setup at the beginning of the livebook. There is no possibility of even putting the markdown cell before it ;/

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd (hunk 2 above)
Comment (Janix4000) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:61]: Added reference

Comment (Janix4000) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:61]: I cannot use the `@` syntax outside the module, but I created a corresponding constant variable.

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+Run pipeline:
+
+```elixir
+alias Membrane.RemoteControlled, as: RC
+
+pipeline = RC.Pipeline.start!()
+RC.Pipeline.exec_actions(pipeline, spec: structure, playback: :playing)
+```
```
[the remark this answers is not in the document]
Comment (Janix4000) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:89]: It is an issue related to the Kino package, not to this specific example.

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd (hunk 1 above)
Comment (WojciechBarczynski) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:23]: Ok

File: livebook_examples/playing_mp3_file/playing_mp3_file.livemd (hunk 3 above)
[the remark this answers is not in the document]
Comment (WojciechBarczynski) [reply livebook_examples/playing_mp3_file/playing_mp3_file.livemd:92]: Write an issue for that

**Review by WojciechBarczynski — APPROVED** [review]

**Pull request merged**